### ISSUE TITLE
The Nepali calendar is off by one day.

### ISSUE THREAD

**Issue by FosRexx** [persian-calendar/persian-calendar#1354]
The Nepali calendar(Vikram Samvat) in the app is one day early because the calendar in the app missed a day in the previous month i.e. जेष्ठ ३२ (Jestha 32).

The second month(जेष्ठ, Jestha) of this year(2081) in the Nepali calendar has 32 days but in the app it only has 31 days, this is causing the calendar to be off by 1 day.

**Comment by ebraminio** [comment]
Hi. First of all, thanks for the interest, it's exciting to me that you even have found this app 😊 I used this library as the basis of my implementation https://github.com/techgaun/ad-bs-converter and there is a bug report there about inaccuracy, I should find some other source to improve the implementation and I hope the issue won't persist till the next month, sorry again and thanks 😊

**Comment by FosRexx** [comment]
Hello again,

Umm, looks like this particular problem is resolved. Just like the 2nd month of the Nepali calendar **in the app** missing a day i.e. जेष्ठ ३२ (Jestha 32). There was an additional extra day in the 3rd month **in the app** i.e. असार 32 (Asar 32). And hence they cancelled each other and now the calendar is in sync. 

But only for a couple of months, because the same problem will happen in the 11th and 12th month of this year. In the official Nepali calendar the 11th month फाल्गुन (Falgun) has 29 days but the app shows it has 30 days and again in the 12th month चैत्र (Chaitra), we have 31 days but the app shows only 30. 

In summary the app has an additional extra day in the 11th month i.e. फाल्गुन ३० (Falgun 30). And missing a day in the 12th month i.e. चैत्र ३१ (Chaitra 31). The year that I am talking about is २०८१ (2081) in Vikram Samvat.

**Comment by ebraminio** [comment]
Actually I've found a solution this time, https://github.com/persian-calendar/calendar/commit/5eb54a517c1fbe66065e2b3e4ed664b18929c912 which has fixed 2081 and 2082! This will be on the next release of the app which happen as soon some of our beta dependencies from Google turn to stable (I guess will be on some weeks)
